src/index.test.ts

```diff
@@ -1,40 +1,62 @@
 import { maxParallel, scan, scanXmlReport } from "."
 
 const root = "/root/"
+const expectedAndroidLintViolation1: Violation = {
```

Comment:
Extracting these into `const` to have the fields available later for assertions to avoid duplicating it everywhere.